packages/codemod/src/test/__testfixtures__/replace-zod-import-with-v3.output.ts

```diff
@@ -0,0 +1,21 @@
+// @ts-nocheck
+import { z } from 'zod/v3';
+import { generateText } from 'ai';
+
+const schema = z.object({
+  name: z.string(),
+  age: z.number(),
+});
+
+const result = await generateText({
+  model: openai('gpt-4'),
+  prompt: 'Generate a person',
+  schema,
+});
+
+// Other imports should not be affected
+import x from 'other-package';
+import { y } from 'zod';
+
+// Default import with different name should not be transformed
+import zod from 'zod';
```

Comment:
I'm not sure about these two. Are there other imports from `zod` that should now be imported form `zod/v3` - can you please double check if you haven't yet?

If we only want to import `z` from `zod/3`, but there are other imports that people should keep importing from `zod`, then I'd cover that case to fixtures. It would need to split up a single import into 2

Same about the default import. Is that even a thing that is possible right now?